content/en/docs/languages/go/instrumentation.md

```diff
@@ -876,7 +875,158 @@ meterProvider := metric.NewMeterProvider(
 
 ## Logs
 
-The logs API is currently unstable, documentation TBA.
+Logs are distinct from metrics and traces in that **there is no user-facing
+OpenTelemetry logs API**. Instead, there is tooling to bridge logs from existing
+popular log packages (e.g. slog, logrus, zap, logr) into the OpenTelemetry
+ecosystem. For rationale behind this design decision, see
+[Logging specification](/docs/specs/otel/logs/).
+
+The two typical workflows discussed below each cater to different application
+requirements.
+
+### Direct to collector (Experimental)
```

Comment:
```suggestion
### Direct to Collector (experimental)
```

Reply:
Same is in other places. Should I fix it as well?

Reply:
I am not sure if this is a good proposal. E.g.
- https://opentelemetry.io/docs/specs/semconv/resource/
- https://opentelemetry.io/docs/languages/go/exporters/

Reply:
I created https://github.com/open-telemetry/opentelemetry.io/issues/4488 to track this.